docsrc/conf.py

```diff
@@ -163,7 +163,7 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = { 'github_version': 'master'}
+html_theme_options = { }
```

Comment:
Ooooooh interesting, what consequence does this discarding this detail have?  The stable branches all have `'github_version': 'cyrus-imapd-3.10'` or whatever here.

Reply:
I believe this is now redundant to the `html_context` configuration, which is already present in conf.py.  I've made internal issue CYR-1481 to investigate this and fix if needed